features/custom_matchers/define_diffable_matcher.feature

```diff
@@ -27,3 +27,49 @@ Feature: define diffable matcher
              +two
               lines
       """
+
+  Scenario: Redefine actual
+
+    Sometimes is neccessary to overwrite actual to make diffing work, e.g. if
+    `actual` is a name of a file you want to read from. For this to work you
+    need to overwrite `@actual` in your matcher.
```

Comment:
@myronmarston Do you have a "test" publisher available on relish where we can try to release the new version to see if it looks good with documentation after the scenario-title?